undefined: "go.opencensus.io/trace".NewSpan

```
# github.com/go-kit/kit/tracing/opencensus
src/github.com/go-kit/kit/tracing/opencensus/grpc.go:39:12: undefined: "go.opencensus.io/trace".NewSpan
src/github.com/go-kit/kit/tracing/opencensus/http.go:41:12: undefined: "go.opencensus.io/trace".NewSpan
```

`NewSpan` was deprecated and has been removed recently.

Build breakage found by [smallrepo.com](https://smallrepo.com).  See the [build log](https://smallrepo.com/builds/20180709-192947-14ceb3aa) for more details.

--- a/tracing/opencensus/grpc.go
+++ b/tracing/opencensus/grpc.go
@@ -37,12 +37,10 @@ func GRPCClientTrace(options ...TracerOption) kitgrpc.ClientOption {
 			}
 
 			ctx, span := trace.StartSpan(
+				ctx,
 				name,
-				trace.FromContext(ctx),
-				trace.StartOptions{
-					Sampler:  cfg.Sampler,
-					SpanKind: trace.SpanKindClient,
-				},
+				trace.WithSampler(cfg.Sampler),
+				trace.WithSpanKind(trace.SpanKindClient),
 			)
 
 			if !cfg.Public {

--- a/tracing/opencensus/http.go
+++ b/tracing/opencensus/http.go
@@ -39,12 +39,10 @@ func HTTPClientTrace(options ...TracerOption) kithttp.ClientOption {
 			}
 
 			ctx, span := trace.StartSpan(
+				ctx,
 				name,
-				trace.FromContext(ctx),
-				trace.StartOptions{
-					Sampler:  cfg.Sampler,
-					SpanKind: trace.SpanKindClient,
-				},
+				trace.WithSampler(cfg.Sampler),
+				trace.WithSpanKind(trace.SpanKindClient),
 			)
 
 			span.AddAttributes(

